client/reader/following-edit/empty.jsx

```diff
@@ -0,0 +1,46 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import EmptyContent from 'components/empty-content';
+import i18n from 'lib/mixins/i18n';
+import { recordAction, recordGaEvent } from 'reader/stats';
+
+const FollowingEditEmptyContent = React.createClass( {
+	recordAction() {
+		recordAction( 'clicked_discover_on_empty' );
+		recordGaEvent( 'Clicked Discover on EmptyContent' );
+	},
+
+	recordSecondaryAction() {
+		recordAction( 'clicked_recommendations_on_empty' );
+		recordGaEvent( 'Clicked Recommendations on EmptyContent' );
+	},
+
```

Comment:
could we push a track on componentDidMount too? I'd like to see how many folks are seeing this message.